package.json

```diff
@@ -30,7 +30,8 @@
     "lodash": "~3.10.0",
     "node-uuid": "~1.4.0",
```

Comment:
We are already using node-uuid in this project. IIRC it does basically the same thing as uuid. I believe we use node-uuid in other projects as well